lesson06/assignment/index.html

```diff
@@ -1,15 +1,132 @@
+            <button type="button" onclick="daysVisiting()">submit</button>
+            <p id="thankYou"></p>
+        </section>
+
+        <section class="ps-4">
+            <h2 class="fs-4 ps-2 fst-italic pb-3">Select and item to see areas in Washington and Oregon</h2>
+            <div class="accordion accordion-flush p-1 w-50" id="accordionFlushElement">
+                <div class="accordion-item border border-primary border-2">
+                    <h2 class="accordion-header" id="flush-headingOne">
+                        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+                            data-bs-target="#flush-collapseOne">
+                            Select here, to see places to roam while your in Washington.
+                        </button>
+                    </h2>
+                    <div id="flush-collapseOne" class="accordion-collage collapse"
+                        data-bs-parent="#accordionFlushElement">
+                        <div class="accordion-body lh-base">Enjoy locations in Washington State. Head West and you'll
+                            find
+                            the Ocean. Head East and you'll find the Cascade Mountain Range, Head further East and
+                            you'll
+                            find the plateau. Head SouthEast past the mountains and you'll meet Wine Country.</div>
+                    </div>
+                </div>
+                <div class="accordion-item border border-secondary border-2">
+                    <h2 class="accordion-header" id="flush-headingTwo">
+                        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+                            data-bs-target="#flush-collapseTwo">
+                            Select here, to see places to roam while your in Oregon
+                        </button>
+                    </h2>
+                    <div id="flush-collapseTwo" class="accordion-collapse collapse"
+                        data-bs-parent="#accordionFlushElement">
+                        <div class="accordion-body lh-base">Enjoy locations in Oregon State. From the Northern Border on
+                            Interstate 5. Head West and you'll find the Ocean. Head East and you'll be driving up the
+                            Columbia River, with numerous dam's. Head further east and it will be plateau as can be
+                            found in
+                            Washington. Head South and you'll find many beautiful areas to visit. Table Rock, Famous
+                            tall flat rock special to Indigenous Peoples of Oregon.</div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="pt-4">
+            <h2 class="fs-4 ps-4">Please share your email with us for our mailing list.</h2>
+            <form class="ps-4 pb-5" name="vacation">
+                <div class="mb-3">
```

Comment:
Remember, your `<form>` should contain all the form elements, including your checkboxes and other form content.